mongo-init.js

```diff
@@ -0,0 +1,5 @@
+db.createCollection('users');
```

Comment:
Do we need to also create the wondervilleActivityBoard database in this script, or is that already handled elsewhere?

Reply:
That's done on line 42 of the yml file.
`MONGO_INITDB_DATABASE: ${MONGODB_DB_NAME}`

Reply:
Right right, sounds good